### PR TITLE
Fix VFIO CI

### PR DESCRIPTION
**Pull request by devimc**

### CODE REVIEW

File: .ci/vfio_jenkins_job_build.sh [hunk 1]
```diff
@@ -70,6 +72,13 @@ create_user_data() {
 		dnf_proxy="dnf:
   https_proxy: ${https_proxy}
   proxy: ${http_proxy}"
+		docker_proxy='[Service]
+    Environment="HTTP_PROXY='${http_proxy}'" "HTTPS_PROXY='${https_proxy}'" "NO_PROXY='${no_proxy}'"'
```
Comment (chavafg) [inline .ci/vfio_jenkins_job_build.sh:76]: indentation issue?

Comment (devimc) [reply .ci/vfio_jenkins_job_build.sh:76]: nop, the yaml where this variable is used requires these spaces

**Pull request merged**